### PR TITLE
Interactive to-do list

### PR DESCRIPTION
**Pull request by 2004-okari**
1. Added event listener to the checkbox (change).
2. Updated items object's value for completed key upon user actions.
3. Implemented a function for the "Clear all completed" button (using filter() method).
4 Stored the updated array of items in local storage, so the user gets the correct list of values after the page reloads.

### CODE REVIEW

File: src/crud.js [hunk 1]
```diff
@@ -43,10 +44,41 @@ const renderTasks = () => {
...
+  checkBoxes.forEach((box, index) => {
+    box.addEventListener('change', () => {
+      if (box.checked) {
+        tasks[index].completed = true;
+      } else {
+        tasks[index].completed = false;
+      }
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    });
+  });
```
Comment (Abenezer-Tilahun) [inline src/crud.js:59]: - [ ] As stated in the project requirement Kindly make sure that you **Add a new JavaScript file and import it as a module: that contain methods related to the status updates (completed: true / false).**kindly Fix it.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @2004-okari  , 

Great job for implemented  the some of the Project Requirement , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linter errors
✔️ Correct Git Hub-Flow
✔️ Professional README

almost

## Required Changes ♻️
_Check the comments below the review._

- [ ] Kindly make sure that your Commit messages should be meaningful that describe the change you pushed to the PR this helps the other collaborators and reviewer understand your changes and makes your PR professional.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/crud.js (hunk 1 above)
Comment (peterrobert) [reply src/crud.js:59]: This change has been done well

Comment (2004-okari) [reply src/crud.js:61]: please expound

**Review by peterrobert — APPROVED** [review]
Hi @2004-okari , 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**